src/components/forms/ScmToolSelect.js

```diff
@@ -14,37 +14,34 @@ export const ScmToolSelect = ({
   helpText = SCM_SUPPORT_HELP_TEXT,
   showProductPrompts = true,
 }) => {
-  const { accent, border, placeholder, background, text, label: labelStyle } = INPUT_COLORS[color];
-
-  function onInputChange(e) {
-    const newScmTool = { ...SCM_TOOLS.find(({ value }) => value === e.target.value) };
-    onChange(newScmTool);
-  }
+  const { label: labelStyle } = INPUT_COLORS[color];
 
   return (
     <>
       <label className={`block text-lg font-medium ${labelStyle}`} htmlFor="scm">
         {label}
       </label>
 
-      <select
-        className={`block mt-1.5 appearance-none w-full py-3 px-4 block shadow-sm rounded-md ${background} ${text} ${accent} ${border} ${placeholder}`}
-        id="scm"
-        value={currentValue.value}
-        name="scm"
-        onChange={onInputChange}
-      >
-        {SCM_TOOLS.map(({ value, label }) => (
-          <option
-            key={`sct-option-${value}`}
-            value={value}
-            name={`sct-option-${value}`}
-            id={`${idPrefix}scm-${value}-input`}
-          >
-            {label}
-          </option>
-        ))}
-      </select>
+      <div className="mt-1.5">
+        <Select
+          value={currentValue.value}
+          options={SCM_TOOLS}
+          onChange={onChange}
+          id="scm"
+          name="scm"
+        >
+          {SCM_TOOLS.map(({ value, label }) => (
+            <option
+              key={`sct-option-${value}`}
+              value={value}
+              name={`sct-option-${value}`}
+              id={`${idPrefix}scm-${value}-input`}
+            >
+              {label}
+            </option>
+          ))}
+        </Select>
+      </div>
```

Comment:
Extracting a `<Select />` component because I want common functionality for sorting.